spec/dashboard-below-map-view.spec.js

```diff
@@ -23,7 +23,7 @@ describe('dashboard-below-map-view', function () {
       timeSeriesWidgetModelFake.dataviewModel.getUnfilteredDataModel = function () {
         return new Backbone.Model();
       };
-      timeSeriesWidgetModelFake.dataviewModel.getData = function () {
+      timeSeriesWidgetModelFake.dataviewModel.getUnfilteredData = function () {
```

Comment:
some occurences of `unfilteredData` have been rephrased into `originalData` this affects cartodb.js too, but shall we take care of every time it appears?